graylog2-server/src/test/java/org/graylog2/alerts/FormattedEmailAlertSenderTest.java

```diff
@@ -240,4 +247,21 @@ public void defaultBodyTemplateShowsBacklogIfBacklogIsNotEmpty() throws Exceptio
                         "Last messages accounting for this alert:\n",
                         message.toString());
     }
+
+    @Test
+    public void buildBodyEscapeHtmlOrNot() throws Exception {
+        Engine templateEngine = new JmteEngineProvider(new HashSet<NamedRenderer>()).get();
+        FormattedEmailAlertSender emailAlertSender = new FormattedEmailAlertSender(new EmailConfiguration(), mockNotificationService, mockNodeId, templateEngine);
+        Configuration pluginConfig = new Configuration(Collections.<String, Object>singletonMap("body", "Test: ${stream.title;htmlsafe}${stream.title}"));
```

Comment:
Hi @zhu  
Thank you for this PR. Much appreciated. And sorry for the late response.
I'm wondering why you wanted to use a `NamedRenderer` here. That would require everyone to update their email templates.
Wouldn't it be better to always escape the values in the context of an html email?

Reply:
Hi @mpfz0r 
Yes, it is. I just make an update.

Reply:
Thank you :pray: 

I made a new PR based on your changes: https://github.com/Graylog2/graylog2-server/pull/14286